cypress/integration/plugins/security-dashboards-plugin/sanity_tests.spec.js

```diff
@@ -0,0 +1,199 @@
+        .eq(0)
+        .type('indices:data/read/msearch')
+        .type('{downArrow}{enter}');
+
+      cy.get('input[data-test-subj="comboBoxSearchInput"]')
+        .eq(2)
+        .type('indices:data/read/search*')
+        .type('{downArrow}{enter}');
+
+      cy.get('#index-input-box').type(
+        'opensearch_dashboards_sample_data_flights*'
+      );
+
+      // Set Document Level Security Query
+      cy.get('textarea').type(documentLevelSecurityQuery);
+
+      // Anonymize fields
+      cy.get('input[data-test-subj="comboBoxSearchInput"][role="textbox"]')
+        .eq(4)
+        .type('FlightNum');
+
+      // Add Tenant Permissions
+      cy.get('input[id="roles-tenant-permission-box"]')
+        .type(tenantName)
+        .type('{enter}');
+
+      // Save Role Definition
+      cy.get('button').contains('Create').click();
+
+      // Verify that the role is created
+      cy.contains(roleName).should('exist');
+    });
+
+    it('should add a new role mapping', () => {
+      // Navigate to Role Mappings
+      cy.visit(
+        `${BASE_PATH}/app/security-dashboards-plugin#/roles/edit/${roleName}/mapuser`
+      );
+
+      cy.get('div[data-test-subj="comboBoxInput"]').type(username);
+      cy.get('button[id="map"]').click();
+    });
+
+    it('should create a new index pattern', () => {
+      cy.visit(`${BASE_PATH}/app/home?security_tenant=${tenantName}`);
+      cy.visit(`${BASE_PATH}/app/home#/tutorial_directory/sampleData`, {
+        retryOnStatusCodeFailure: true,
+      });
+
+      createSampleFlightDataIfDoesntExist();
+      // Step 3: Navigate to Manage data to add an index pattern
+      cy.visit(`${BASE_PATH}/app/home`);
+      cy.get('button[aria-label="Closes this modal window"]').click();
+      cy.get('a').contains('Manage').click(); // Adjust the selector as needed
+
+      // Step 4: Add the index pattern
+      cy.get('[data-test-subj="indexPatterns"]').click();
+      cy.get('[data-test-subj="createIndexPatternButton"]').click();
+      cy.get('input[data-test-subj="createIndexPatternNameInput"]').type(
+        indexPattern
+      );
+      cy.get(
+        'button[data-test-subj="createIndexPatternGoToStep2Button"]'
+      ).click();
+
+      // Assuming a timestamp field needs to be selected
+      cy.get(
+        'select[data-test-subj="createIndexPatternTimeFieldSelect"]'
+      ).select('timestamp');
+
+      cy.get('option[value="timestamp"]');
+      cy.get('button[data-test-subj="createIndexPatternButton"]').click();
+    });
+  });
+}
+
+function createSampleFlightDataIfDoesntExist() {
+  cy.get('body').then(($body) => {
+    if ($body.find('[data-test-subj="addSampleDataSetflights"]').length > 0) {
+      // If the element exists, click on it
+      cy.get('[data-test-subj="addSampleDataSetflights"]').click();
+    } else {
+      // The element does not exist, you can log a message or take other actions
+      cy.get('[data-test-subj="launchSampleDataSetflights"]').click();
+    }
+  });
```

Comment:
this can be simplified by https://github.com/opensearch-project/opensearch-dashboards-functional-test/blob/main/cypress/utils/commands.js#L526

Reply:
I've replaced it with the command